xlators/mgmt/glusterd/src/glusterd.h

```diff
@@ -803,6 +803,96 @@ typedef ssize_t (*gd_serialize_t)(struct iovec outmsg, void *args);
                                                                                \
     } while (0)
 
+struct opval_dep;
+typedef struct opval_dep opval_dep_t;
+
+typedef enum { OP_OFF = 0, OP_ON, CLAUSE_LABEL } optn_value;
```

Comment:
Please use an unified prefix for the enum values. If you find `OP_CLAUSE_LABEL` too much mouthful, `OP_LABEL` would be fine too.